src/nix/run.md

```diff
@@ -35,7 +35,7 @@ R""(
 
 # Description
 
-`nix run` builds and runs *installable*, which must evaluate to an
+`nix run` builds and runs [*installable*](./nix.md#installables), which must evaluate to an
 *app* or a regular Nix derivation.
```

Comment:
I know it would be nice changing it while we're at it, but I'd like avoid scope creep to reduce the time in flight of PRs, therefore I'd ask you to make a separate PR to address this you want it fixed.